EddieHubCommunity/BioDrop
# Feat middleware to protect auth and admin pages

## Fixes Issue

closes #8389 

Finalising PR https://github.com/EddieHubCommunity/BioDrop/pull/9249

## TO TEST

| LOGGED IN | ACTION | EXPECTED RESULT |
| :--- | :--- | :--- |
| false | try to access protected pages like `/account/onboarding` | page not accessible, redirected |
| true | try to access admin pages `/admin/statistics` |  page not accessible, redirected |





## Changes proposed






## Check List (Check all the applicable boxes) 

- [ ] My change requires changes to the documentation.

## Screenshots



## Note to reviewers
